src/main/java/io/vertx/ext/mongo/impl/config/CompressorListParser.java

```diff
@@ -0,0 +1,77 @@
+package io.vertx.ext.mongo.impl.config;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoCompressor;
+import com.mongodb.lang.Nullable;
+import io.vertx.core.json.JsonObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompressorListParser {
+
+  private final List<String> compressorKeys;
```

Comment:
It's not necessary to create at runtime a list of keys for config. Instead, we can do with a couple of `if` statements in the `compressorList` method